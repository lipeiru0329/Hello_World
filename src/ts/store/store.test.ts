import store from './store';

describe('store', () => {
	test('actions', () => {
		return new Promise(resolve =>
			setTimeout(() => {
				expect(store.getState()).toMatchSnapshot();
				resolve();
			}, 1000)
		);
	});
});
