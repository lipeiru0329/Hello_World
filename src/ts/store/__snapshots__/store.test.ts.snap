// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store actions 1`] = `
Object {
  "firebase": Object {
    "auth": false,
  },
}
`;
